azure-functions/acbs-function/src/functions/acbs-amend-facility-master-record.js

```diff
@@ -32,53 +32,53 @@ df.app.orchestration('acbs-amend-facility-master-record', function* amendFacilit
   const payload = context.df.input;
 
   try {
-    if (payload) {
-      const { deal, facilityId, fmr, etag, amendments } = payload;
-      const { amendment } = amendments;
-      let facilityMasterRecordAmendments;
+    if (!payload) {
+      throw new Error('Invalid argument set provided');
```

Comment:
Might be worth applying specific error message here as well please. Thanks

Reply:
I'll commonise this with `Invalid payload provided`